google/generativeai/files.py

```diff
@@ -105,6 +123,10 @@ def get_file(name: str) -> file_types.File:
     return file_types.File(client.get_file(name=name))
 
 
+async def get_file_async(*args, **kwargs):
+    return await asyncio.to_thread(get_file, *args, **kwargs)
```

Comment:
We have an async client, we should use it. Let's copy-paste the code, and add the missing `async`s